### ISSUE TITLE
Rsnapshot version

### ISSUE THREAD

**Issue by nsasia** [rsnapshot/rsnapshot#227]
I can't get version of rsnapshot by "rsnapshot --v".
Is there way I can see version of rsnapshot ?

**Comment by StefanMoch** [comment]
`rsnapshot version` should print the version.

**Comment by nsasia** [comment]
![image](https://user-images.githubusercontent.com/51106137/58450921-70fa0d80-813b-11e9-8aa7-c5618cbe623d.png)
It showed "rsnapshot" without version.

**Comment by StefanMoch** [comment]
> It showed "rsnapshot" without version.

If you installed rsnapshot via your OS package manager, you may
query it for the package version number.

For apt/dpkg on Debian, Ubuntu, etc.:
`dpkg -l | grep rsnapshot`

For rpm based distributions:
`rpm -qa | grep rsnapshot`

The package may contain additional patches, so the version
number can be misleading sometimes.

**Comment by davidcantrelluk2** [comment]
`rsnapshot version` has been a thing for *years*, so I'm surprised it doesn't show anything more helpful. Can you show us the output from `head -70 \`which rsnapshot\``.

**Comment by nsasia** [comment]
This what it showed.
`root@ao-server:~# head -70 \which rsnapshot
head: cannot open 'which' for reading: No such file or directory
head: cannot open 'rsnapshot' for reading: No such file or directory
`

**Comment by davidcantrelluk2** [comment]
Sorry, Github isn't very good at formatting code in comments. I've edited it to make it show correctly what I wanted, so please try again!

**Comment by nsasia** [comment]
[rsnapshot_head.txt](https://github.com/rsnapshot/rsnapshot/files/3227102/rsnapshot_head.txt)
Here it is

**Comment by davidcantrelluk2** [comment]
And there's the problem: `my $VERSION = '';`. [Here's that line in the git repository](https://github.com/rsnapshot/rsnapshot/blob/master/rsnapshot-program.pl#L66). The autoconf witchcraft that sets it appears to have broken. It's not terribly important, but assuming that you're using a vaguely recent copy from git, just set it to 1.3.2.

**Comment by sam-at-github** [comment]
Confirmed issue on `Ubuntu 18.04.2 LTS` and `Debian GNU/Linux 9.9 (stretch)`, both of which have packaged `1.4.2-1`. Build and install with `make` works but gives "rsnapshot 1.4.2-18-g717d75a". Maintainer (Guillaume Delacour in both cases) has probably tried to change this for to a number consistent with packaging guidelines(?) I submitted bur report to [Debian tracker](https://bugs.debian.org/cgi-bin/pkgreport.cgi?dist=unstable;package=rsnapshot).
